src/Store.ts

```diff
@@ -0,0 +1,13 @@
+import { create } from 'zustand';
```

Comment:
:+1: If you add any npm library you should list it in the package.json, in the `dependencies` array

generally, if you run the command like this `npm install zustand` it should happen automatically

Reply:
but it has not happened